platform/core/src/services/DicomMetadataStore/DicomMetadataStore.js

```diff
@@ -211,7 +211,15 @@ const BaseImplementation = {
     let study = _getStudy(StudyInstanceUID);
     if (!study) {
       study = createStudyMetadata(StudyInstanceUID);
+      // Will typically be undefined with a compliant DICOMweb server, reset later
       study.StudyDescription = seriesSummaryMetadata[0].StudyDescription;
+      study.ModalitiesInStudy = [];
+      seriesSummaryMetadata.forEach(item => {
+        if (study.ModalitiesInStudy.indexOf(item.Modality) == -1) {
+          study.ModalitiesInStudy.push(item.Modality);
+        }
+      });
+      study.NumberOfStudyRelatedSeries = seriesSummaryMetadata.length;
```

Comment:
same here, it should be defined in the `create...`

Also isn't `study.series.length` the same length as `NumberOfStudyRelatedSereis`? why do you need to define a new one?

Reply:
Because NumberOfStudyRelatedSeries is the formal DICOM name, and the metadata is supposed to mirror the DICOM names, it is just that the particular loading strategy chosen doesn't allow getting the value.